### PR TITLE
Add expansion inf

### PR DESCRIPTION
**Pull request by meghnasiingh**
Signed-off-by: Meghna Singh <Meghna.Singh1@ibm.com>

### CODE REVIEW

File: lib/provider/data_types.go [hunk 1]
```diff
@@ -136,3 +136,16 @@ type VolumeList struct {
...
+// ExpandVolumeRequest ...
+type ExpandVolumeRequest struct {
+	// VolumeID id for the volume
+	VolumeID string `json:"volumeID"`
+
+	// changed Volume name
+	Name *string `json:"name,omitempty"`
+
+	// The new Capacity of the volume, in GiB
+	//Capacity *int `json:"capacity"`
```
Comment (arahamad) [inline lib/provider/data_types.go:149]: remove un-used or commented code

**Review by arahamad — COMMENTED** [review]
LGTM, just removed un-used or commented code

**Review by arahamad — APPROVED** [review]
LGTM

**Review by akgunjal — APPROVED** [review]
LGTM

**Pull request merged**